syntax = "proto3";
package movie_reviews;

// Next ID: 3
message Critic {
  string full_name = 1;
  string publisher = 2;
}
